python/lsst/validate/drp/validate.py

```diff
@@ -202,10 +200,10 @@ def runOneRepo(repo, dataIds=None, metrics=None, outputPrefix='', verbose=False,
     jobs = {}
     for filterName in allFilters:
         # Do this here so that each outputPrefix will have a different name for each filter.
-        if outputPrefix is None:
+        if outputPrefix is None or outputPrefix == '':
```

Comment:
I believe the `None` test could be omitted if the `#run` was using `""` as the default value for `outputPrefix`.